packages/flutter/test/material/app_test.dart

```diff
@@ -25,6 +25,23 @@ class StateMarkerState extends State<StateMarker> {
   }
 }
 
+class PopWidget extends StatelessWidget {
+  final dynamic value;
+  final String name;
+  final BuildContext context;
+
+  const PopWidget(this.context, this.name, this.value);
```

Comment:
passing a context to a widget is pretty crazy. Maybe instead of a PopWidget, use a Builder?